test/objects.test.js

```diff
@@ -330,9 +330,28 @@ describe('objects', function() {
     async.parallel([
       function(done) {
         parse.createObjects(className, dogs, function(err, res, body, success) {
-          success.should.be.true;
+          success.should.be.true;          
           should.not.exist(err);
-
+          //parse error is set in body.error          
+          /*
+            it may failed by body.code 
+            002 the service is currently unavailable
+            124 timeout
+            154 out of count requestlimit
+            155 out of normal requestlimit
+            159 temporary error
+          */
+          var retryable = [ 2, 124, 154, 155, 159];
+          if(body.code && retryable.indexOf(body.code) > -1){
+            console.log('code', body.code);
+            console.log('2', 'the service is currently unavailable');
```

Comment:
Sorry I could not quite figure out what this is trying to test. Would you please explain? :)